pkgs/development/python-modules/clikit/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi,
+  isPy27, isPy34,
+  pylev, pastel, typing, enum34 }:
+buildPythonPackage rec {
+  pname = "clikit";
+  version = "0.2.3";
+
+  src = fetchPypi {
```

Comment:
I'm also not seeing any tests in this tarball also. So checks should be disabled.

Reply:
Although for `pylev` getting the GitHub source worked so we could run tests because it uses `setup.py`, for `clikit` I'm running into a chicken-egg problem because it wants to use `poetry` as its build system to build from `pyproject.toml`. Thoughts?

My instinct is we bootstrap just by not using tests, and then somehow use that that `poetry` to rebuild its dependencies with tests, but I'm not sure how to do that.